### PR TITLE
Ashley/restructure v.0

### PR DESCRIPTION
**Pull request by yeseulan**
Restructure code for feature/uploadfile and feature/createuserfolder in order to proceed to unit-testing. (Codes adjust possible changes based on the review.)

### CODE REVIEW

File: service.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+import hwsc_file_transaction_svc_pb2_grpc
+
+class FileTransactionService(hwsc_file_transaction_svc_pb2_grpc.FileTransactionServiceServicer):
+     def __init__(self):
+
+         class Servicer(hwsc_file_transaction_svc_pb2_grpc.FileTransactionServiceServicer):
+             def __init__(self):
+                 pass
+
+             def GetStatus(self, request, context):
```
Comment (faraonc) [inline service.py]: add `# TODO` here

File: service.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                 print("Get Status")
+
+             def DownloadZippedFiles(self, request_iterator, context):
```
Comment (faraonc) [inline service.py]: add `#TODO` here

File: hwsc_file_transaction_svc_pb2_grpc.py [hunk 3]
```diff
@@ -49,7 +54,14 @@ def UploadFile(self, request_iterator, context):
     context.set_details('Method not implemented!')
     raise NotImplementedError('Method not implemented!')
 
-  def DownloadFile(self, request, context):
+  def DownloadZippedFiles(self, request, context):
+    # missing associated documentation comment in .proto file
```
Comment (faraonc) [inline hwsc_file_transaction_svc_pb2_grpc.py]: Can you investigate this?

File: test_utility.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import pytest
+import utility
+
+# testing with already existing uuid
```
Comment (faraonc) [inline test_utility.py]: still need a better way to unit test

File: utility.py [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+            d['uuid'] = property.uuid
+        if len(property.file_name) > 1:
+            d['f_name'] = property.file_name
+        if len(property.buffer) > 1:
+            stream.write(property.buffer)
+        else:
+            pass
+
+        d['stream'] = stream
+    return d
```
Comment (faraonc) [inline utility.py]: Missing documentation on all functions in accordance with https://www.python.org/dev/peps/pep-0257/

File: test_utility.py (hunk 4 above)
Comment (yeseulan) [reply test_utility.py]: U mean having multiple parameters in one function? Regarding that, I will implement and make a new pull request.

File: utility.py (hunk 5 above)
Comment (yeseulan) [reply utility.py]: Is this relate to this comment `# missing associated documentation comment in .proto file` ?

Comment (faraonc) [reply utility.py]: Yes the comments in the proto file are used to document the resulting source code from the compilation.

File: test_utility.py (hunk 4 above)
Comment (faraonc) [reply test_utility.py]: Please write a ` #TODO`

File: utility.py (hunk 5 above)
Comment (faraonc) [reply utility.py]: No. This is not generated by the proto file. This is your own source code.

File: utility.py [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+    #TODO
+    #ADD try-catch block
+
+    # Checking whether the blobs that associated with uuid exists
+    if has_folder:
+        container_type = get_file_type(file_name)
+        container_name = uuid + '-' + container_type
+
+        # Set the permission so the blobs are public.
+        block_blob_service.set_container_acl(container_name, public_access=PublicAccess.Container)
```
Comment (kimlisa) [inline utility.py]: what happens if this fails? i'm assuming the try catch block will catch it? (i know its todo)

**Pull request merged**